cmd/krel/cmd/release_notes.go

```diff
@@ -85,6 +90,38 @@ func init() {
 		"version tag for the notes",
 	)
 
+	releaseNotesCmd.PersistentFlags().StringVarP(
+		&releaseNotesOpts.draftOrg,
+		"draft-org",
+		"",
+		"",
+		"a Github organization or user where the Release Notes PR will be created",
+	)
+
+	releaseNotesCmd.PersistentFlags().StringVarP(
+		&releaseNotesOpts.draftRepo,
+		"draft-repo",
+		"",
+		"",
+		"the name of a Github repository where the Release Notes PR will be created",
+	)
+
+	releaseNotesCmd.PersistentFlags().BoolVarP(
+		&releaseNotesOpts.createDraftPR,
+		"create-draft-pr",
+		"",
+		false,
+		"create the Release Notes draft PR. --draft-org and --draft-repo muste be set along with this option",
```

Comment:
Typo: "muste" -> "must"
```suggestion
		"create the Release Notes draft PR. --draft-org and --draft-repo must be set along with this option",
```

Reply:
Fixed